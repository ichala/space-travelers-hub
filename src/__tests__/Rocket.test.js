import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import RocketProfile from '../Components/Profile/Rockets/Rocket';
import store from '../Redux/configureStore';

describe('Test of Profile Components', () => {
  it('Profile Components Renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <RocketProfile />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
