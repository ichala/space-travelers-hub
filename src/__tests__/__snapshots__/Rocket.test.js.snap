// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of Profile Components Profile Components Renders correctly 1`] = `
<table
  className="zebra"
>
  <thead>
    <tr>
      <th
        className="table-title"
      >
        Rockets
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td
        className="table-title"
      >
        No missions available ..
      </td>
    </tr>
  </tbody>
</table>
`;
